library/src/com/google/maps/android/kml/KmlRenderer.java

```diff
@@ -872,7 +913,27 @@ protected void onPostExecute(Bitmap bitmap) {
     }
 
     /**
-     * Downloads images for use as ground overlays
+     * @param url internet address of the image.
+     * @return the bitmap of that image, scaled according to screen density.
+     */
+    private Bitmap getBitmapFromUrl(String url) throws MalformedURLException, IOException {
+        return BitmapFactory.decodeStream((InputStream) new URL(url).getContent(), null, mBitmapOptions);
+    }
+
+    /**
+     * @param fileName the name of the file to look for within {@link KmlRenderer#mDirectoryName}.
+     * @return the bitmap in the directory {@link KmlRenderer#mDirectoryName} and name fileName, scaled according to screen density;
+     * or null if {@link KmlRenderer#mDirectoryName} is null.
+     */
+    private Bitmap getBitmapFromFile(String fileName) {
+        if (mDirectoryName == null) {
```

Comment:
Why are we not decoding the bitmap from the file if the directory is null? Should be doing it just like the library was previously to support that constructor behavior.